drivers/hmis/app/models/hmis/role.rb

```diff
@@ -93,10 +93,10 @@ def self.permissions_with_descriptions
           'Administration',
         ],
       },
-      can_delete_assigned_project_data: {
-        description: 'Grants access to delete project related data for projects the user can see',
+      can_view_project: {
+        description: 'Grants access to view the project page',
```

Comment:
I'm not sure this description captures what you're doing with can view projects.  Essentially it allows access to the project page, but ALSO is used to limit which enrollment related items a user can see.

Reply:
Updated the description. I do think overall this needs to be described at a higher level on the page, not in each individual permission description. Like, ALL the permissions that are in the "Enrollment" and "Project" category get limited based on two things: (1) which projects are assigned to the collection thats in an ACL with this role, and (2) which of those assigned projects have `can_view_project`.